src/polytope_blmos.jl

```diff
@@ -37,6 +37,44 @@ function is_simple_linear_feasible(sblmo::CubeSimpleBLMO, v)
     return true
 end
 
+function is_decomposition_invariant_oracle_simple(sblmo::CubeSimpleBLMO)
+    lb = sblmo.lower_bounds
+    ub = sblmo.upper_bounds
+    return all(x -> (x[1] == 0 || x[1] == 1) && (x[2] == 0 || x[2] == 1), zip(lb, ub))
```

Comment:
This is not very intuitive. I found the set version that you had first better to read